src/shared/components/DashboardRoute.tsx

```diff
@@ -65,10 +65,10 @@ class DashboardRoute extends PureComponent<Props> {
     }
 
     const params = new URLSearchParams(window.location.search)
-    const vars = (Array.from(params.entries()) || [])
-      .filter(([k, _v]) => /(^vars\[)(.+)(\]$)/g.test(k))
-      .reduce((acc, [k, v]) => {
-        acc[k.slice(5, -1)] = v
+    const vars = Array.from(params.keys())
```

Comment:
none of this was needed, just the dom.iterable changes in the tsconfig. but after i wrote it, i liked it better, so here we are